index.html

```diff
@@ -1,17 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
-    <link rel="stylesheet" href="output/styles.css">
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@200&display=swap');
+    </style>
 </head>
+
 <body>
     <div class="wrapper">
-        <h1 class="name">Santiago Velosa</h1>
-        <h2 class="description">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
-            If you like what you see and have a project you need coded, don’t hesitate and contact me.</h2>
+        <nav class="hamburger">
+            <button type="button" class="btn-icon"></button>
+        </nav>
+        <section class="container">
+            <h1 class="name">Charlie Giorgania</h1>
+            <p class="description">
+                Hello! I am a software developer! I can
+                help you build a product, feature or
+                website. Taka a look of my works.
+                If you like what you see and have a
+                project you need coded, don’t hesitate
+                and contact me.</p>
+            <nav class="button">
+                <button type="button" class="actual-button">
+                    Start collaboration
+                </button>
+            </nav>
```

Comment:
- [ ][OPTIONAL] I think it's better to remove the nav tag and only keep the button. Because the HTML5 semantic tag nav is used to specify a block of navigation links, either within the current document or to other documents. Examples of navigation blocks are tables of contents, menus, and indexes.